image/tarexport/save.go

```diff
@@ -191,15 +189,13 @@ func (s *saveSession) save(outStream io.Writer) error {
 	manifestFileName := filepath.Join(tempDir, manifestFileName)
 	f, err := os.OpenFile(manifestFileName, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0644)
 	if err != nil {
-		f.Close()
 		return err
 	}
+	defer f.Close()
```

Comment:
Okay, in that case @allencloud close the file here explicitly (as well as the defer -- we can ignore the error to close an already-closed file).